fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -405,6 +410,31 @@ public void updateProperties(final IdentifierConverter<Resource, FedoraResource>
         listener.assertNoExceptions();
     }
 
+    private boolean clean(final String updateStmt) {
+        final int start = updateStmt.indexOf("INSERT");
+        final int end = updateStmt.lastIndexOf("WHERE");
+
+        if (start < 0 || end < 0) {
+            return true;
+        }
+
+        final String insertStmt = updateStmt.substring(start, end);
+        final String[] insert = insertStmt.split(".+<[a-zA-Z]*>");
+        int count = 0;
+        final String terminated = "/>";
+
+        for (final String s: insert) {
+            if (s.contains(terminated)) {
```

Comment:
Please add a comment that describes why having a "terminating" token here is an issue.